CHANGELOG.md

```diff
@@ -46,6 +47,9 @@ IMPROVEMENTS:
 * Upgrade Terraform SDK dependency to 0.12.8 [#320]
 * `resource/vcd_vapp_vm` has new field `computer_name` [#334]
 * Import functions can now use custom separators instead of "." [#343]
+* `resource/vcd_independent_disk` Add computed properties (`iops`, `owner_name`, `datastore_name`, `is_attached`, `size_in_bytes`) and full read support [#349]
```

Comment:
size_in_bytes shouldn't be here anymore.

Reply:
updated